### ISSUE TITLE
Support Gab Revisions

### ISSUE THREAD

**Issue by billstclair** [billstclair/elm-mastodon#10]
When you edit a post in Gab's Mastodon fork, it tracks all the revisions. You can get them with:

GET /api/v1/statuses/:id/revisions

(not yet live, but I noticed it in the change log of their repository)

**Comment by billstclair** [comment]
Of course, I also have to figure out the API for making a new revision.

**Comment by asonix** [comment]
actively supporting gab... suddenly I'm less excited about this project

**Comment by billstclair** [comment]
@asonix

GitHub is not the place for political opinions. Please keep those on the fediverse.

You can contact me via my personal Pleroma server, [@billstclair@impeccable.social](https://impeccable.social/billstclair).
